src/main/java/com/iota/iri/network/TipRequester.java

```diff
@@ -0,0 +1,106 @@
+package com.iota.iri.network;
+
+import com.iota.iri.controllers.TransactionViewModel;
+import com.iota.iri.network.neighbor.Neighbor;
+import com.iota.iri.network.pipeline.TransactionProcessingPipeline;
+import com.iota.iri.service.milestone.LatestMilestoneTracker;
+import com.iota.iri.storage.Tangle;
+import com.iota.iri.utils.thread.ThreadIdentifier;
+import com.iota.iri.utils.thread.ThreadUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * The {@link TipRequester} requests tips from all neighbors in a given interval.
+ */
+public class TipRequester {
+
+    private static final Logger log = LoggerFactory.getLogger(TipRequester.class);
+
+    private final AtomicBoolean shutdown = new AtomicBoolean(false);
+
+    private final ThreadIdentifier tipRequesterThreadIdentifier = new ThreadIdentifier("Tip Requester");
+
+    private NeighborRouter neighborRouter;
+    private Tangle tangle;
+    private TransactionRequester txRequester;
+    private LatestMilestoneTracker latestMilestoneTracker;
+
+    /**
+     * Initializes the dependencies.
+     * 
+     * @param neighborRouter         the {@link NeighborRouter} to use
+     * @param tangle                 the {@link Tangle} database to load the latest milestone from
+     * @param latestMilestoneTracker the {@link LatestMilestoneTracker} to gets the latest milestone hash from
+     * @param txRequester            the {@link TransactionRequester} to get the currently number of requested
+     *                               transactions from
+     */
+    public void init(NeighborRouter neighborRouter, Tangle tangle, LatestMilestoneTracker latestMilestoneTracker,
+            TransactionRequester txRequester) {
+        this.neighborRouter = neighborRouter;
+        this.tangle = tangle;
+        this.latestMilestoneTracker = latestMilestoneTracker;
+        this.txRequester = txRequester;
+    }
+
+    /**
+     * Starts a dedicated thread for the {@link TipRequester} and then starts requesting of tips.
+     */
+    public void start() {
+        ThreadUtils.spawnThread(this::requestTips, tipRequesterThreadIdentifier);
```

Comment:
Use the `DedicatedScheduledExecutorService` like in TransactionRequesterWorkerImpl

Reply:
Done.